src/js/tabs/osd.js

```diff
@@ -651,10 +647,8 @@ OSD.loadDisplayFields = function() {
             text: 'osdTextElementArtificialHorizon',
             desc: 'osdDescElementArtificialHorizon',
             defaultPosition() {
+                let position = (OSD.constants.VIDEO_COLS[videoType] >> 1) + ((OSD.constants.VIDEO_ROWS[videoType] >> 1) - 5) * OSD.constants.VIDEO_COLS[videoType] - 1;
                 let position = 74;
```

Comment:
This line overwrites previous initialization

Reply:
Spotted that. Thanks.